app/service/form_service.rb

```diff
@@ -30,4 +30,12 @@ def add_to_default_group!(current_user)
 
     org.default_group.memberships.find_or_create_by!(user: current_user, role: :editor, added_by: current_user)
   end
+
+  def assign_owner!(current_user)
```

Comment:
I'm not sure about this method name, I don't really understand what's happening here from this. Would it also make sense to squash the next commit into this commit, as this commit by itself isn't particularly informative about what's going on here. Looks like we're extracting a method?